motionplan/collision.go

```diff
@@ -312,3 +337,13 @@ func (cs *CollisionSystem) CollisionBetween(keyName, testName string) bool {
 	}
 	return false
 }
+
+// AddCollisionSpecificationToGraphs takes a Collision as an argument and either whitelists collisions between the two entities.
+func (cs *CollisionSystem) AddCollisionSpecificationToGraphs(specification *Collision) error {
```

Comment:
Thinking about this more; I think the reason it seems to make sense to also have a blacklist is that we're using `Collision` objects to represent the input here, which have a field for penetration depth now going unused.

It may make more sense to just have `name1, name2 string` as the parameters for this and the other function